### PR TITLE
Task 02

### PR DESCRIPTION
**Pull request by cwetanow**
#3

### CODE REVIEW

File: 02/Program.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		{
+			var input = Console.ReadLine()
+				.Split(' ')
+				.Select(int.Parse)
+				.ToArray();
+
+			var result = Enumerable.Repeat(1, input.Length)
+				.ToArray();
+
+			for (int i = 0; i < result.Length; i++)
```
[the remark this answers is not in the document]
Comment (cwetanow) [reply 02/Program.cs]: Would you share the solution for O(n)?

[the remark this answers is not in the document]
Comment (maixuanhan) [reply 02/Program.cs]: If you maintain 2 arrays of the sum product of the items from beginning and of the items from ending, you can easily calculate the output array.
For example, [1,2,3,4,5], the 2 array would be [1,2,6,24,120], [5,20,60,120,120]....

[the remark this answers is not in the document]
Comment (cwetanow) [reply 02/Program.cs]: And how would you get the result array without division?

[the remark this answers is not in the document]
Comment (maixuanhan) [reply 02/Program.cs]: image
This is for calculating the number at position 2nd.

[the remark this answers is not in the document]
Comment (maixuanhan) [reply 02/Program.cs]: My implement's here:
https://github.com/maixuanhan/programming-notes/blob/master/dailycodingproblem/problem2/solution.md#solution-2---without-division

**Pull request merged**